Adyen/Core/APIClient/Environment.swift

```diff
@@ -6,42 +6,37 @@
 
 import AdyenNetworking
 import Foundation
-    
-/// :nodoc:
+
 /// Struct that defines the environment to retrieve resources from.
 public struct Environment: AnyAPIEnvironment {
     
     /// :nodoc:
     public var baseURL: URL
 
-    /// :nodoc:
     /// Adyen's test environment.
     public static let test = Environment(baseURL: URL(string: "https://checkoutshopper-test.adyen.com/")!)
 
     /// :nodoc:
     public static let beta = Environment(baseURL: URL(string: "https://checkoutshopper-beta.adyen.com/")!)
-    
-    /// :nodoc:
+
     /// Adyen's default live environment.
-    public static let live = Environment(baseURL: URL(string: "https://checkoutshopper-live.adyen.com/")!)
-    
-    /// :nodoc:
+    @available(*, deprecated, message: "This property no longer supported. Please select matching environment.")
```

Comment:
`"This property no longer supported. Please explicitly select the environment matching your region."`

Reply:
Let's not forget the `is` as in `This property is no longer...`